core/modules/image/image.field.inc

```diff
@@ -737,6 +773,19 @@ function image_field_formatter_settings_summary($field, $instance, $view_mode) {
     $summary[] = t('Loading: @speed', array('@speed' => $settings['image_load']));
   }
 
+  $formatter_settings = $instance['display'][$view_mode]['settings'];
+
+  if (!empty($formatter_settings['image_svg_settings']['width'])
+    || !empty($formatter_settings['image_svg_settings']['width'])) {
+    $summary[] = t(
+      'SVG images attributes: <br /> Width: @widthpx,<br /> Height: @heightpx',
```

Comment:
Lets try this to see if cspell is happy:

```suggestion
      'SVG images attributes: <br /> Width: @width_px,<br /> Height: @height_px',
```

...also, these names do not match the array keys below, so the string replacement won't happen.

Reply:
Fixed